### ISSUE TITLE
Start testrpc with '-i' switch

### ISSUE THREAD

**Issue by area** [sc-forks/solidity-coverage#87]
If we do this, we can set it to 'solidity-coverage' or something similar, and people can use `web3.version.network` in their tests to decide if they're running against coverage if they need to execute slightly differently (most likely, varying gas costs / responses to gas used).

**Comment by cgewecke** [comment]
Agreed - this is a great idea. . .we might also document this in the FAQ gas section since it will  dramatically simplify adding SC to a suite with lots of gas values above the default transaction limit.
